reference/sbom_naming.md

```diff
@@ -0,0 +1,31 @@
+# Best Practices for Naming and Directory Conventions for SBOMs (Software Bill of Materials) in Open Source Projects
+
+Software Bill of Materials (SBOM) plays an essential role in software supply chain transparency. Using SBOM standards such as CycloneDX and SPDX ensures interoperability, accurate dependency tracking, and efficient vulnerability management. Here are some best practices for naming and directory conventions when creating and managing SBOMs.
+
+**1. Consistent Naming Conventions:**
+
+- Use clear, descriptive names that reflect the content of the SBOM. Include the project name, version number, and the SBOM format (CycloneDX or SPDX). For example, `sbom.projectname_v1.2.3_spdx.json` or `sbom.projectname_v1.2.3_cyclonedx.xml`.
+
+- When dealing with multiple SBOMs from different branches or versions of a project, include this detail in the file name. For instance, `sbom.projectname_v1.2.3_beta_cyclonedx.json`.
+
+**2. Directory Structure:**
+
+- Store SBOM files in a dedicated directory, separate from the source code. This might be a top-level directory in the repository named something like `SBOMs`.
+
+**4. Versioning:**
+
+- SBOMs should be version controlled just like source code. Each new release of the software should have a corresponding SBOM.
+
+- To keep track of changes over time, it is good practice to include the version number in both the SBOM filename and inside the SBOM document itself. 
```

Comment:
Changing the name on every check-in feels like it'll lose much of the value of keeping the file in source control in the first place. Are you confident that changing the name frequently brings real value? Thanks